[2023-01-26 18:08:16,747] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-26 18:08:16,761] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-03T00:00:00+00:00 [queued]>
[2023-01-26 18:08:16,761] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 18:08:16,761] {taskinstance.py:1377} INFO - Starting attempt 10 of 12
[2023-01-26 18:08:16,761] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 18:08:16,777] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Respuesta_Confirmacion_NASA> on 2023-01-03 00:00:00+00:00
[2023-01-26 18:08:16,782] {standard_task_runner.py:52} INFO - Started process 933 to run task
[2023-01-26 18:08:16,785] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Respuesta_Confirmacion_NASA', 'scheduled__2023-01-03T00:00:00+00:00', '--job-id', '656', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpx38bg340', '--error-file', '/tmp/tmpqqg_h9wt']
[2023-01-26 18:08:16,786] {standard_task_runner.py:80} INFO - Job 656: Subtask Respuesta_Confirmacion_NASA
[2023-01-26 18:08:16,852] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Respuesta_Confirmacion_NASA scheduled__2023-01-03T00:00:00+00:00 [running]> on host da6de88268cf
[2023-01-26 18:08:16,934] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=frandak2@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Respuesta_Confirmacion_NASA
AIRFLOW_CTX_EXECUTION_DATE=2023-01-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=10
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-03T00:00:00+00:00
[2023-01-26 18:08:16,935] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-26 18:08:16,936] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 20 && echo "ConfirmaciÃ³n de la NASA, pueden proceder" > /tmp/response_20230103.txt']
[2023-01-26 18:08:16,948] {subprocess.py:85} INFO - Output:
[2023-01-26 18:08:36,952] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-26 18:08:37,057] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Respuesta_Confirmacion_NASA, execution_date=20230103T000000, start_date=20230126T180816, end_date=20230126T180837
[2023-01-26 18:08:37,108] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 18:08:37,138] {dagrun.py:1081} ERROR - Failed to get task '<TaskInstance: Respuesta_Nasa.Notificar_Analistas scheduled__2023-01-03T00:00:00+00:00 [removed]>' for dag 'Respuesta_Nasa'. Marking it as removed.
[2023-01-26 18:08:37,175] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
